spirv_glsl.cpp

```diff
@@ -6961,14 +7125,14 @@ void CompilerGLSL::emit_instruction(const Instruction &instruction)
 			//
 			// Including the base will prevent this and would trigger multiple reads
 			// from expression causing it to be forced to an actual temporary in GLSL.
-			auto expr = access_chain_internal(ops[2], &ops[3], length, true, true, &meta);
+			auto expr = access_chain_internal(ops[2], &ops[3], length, true, true, false, &meta);
```

Comment:
I think I'll need to refactor all these bools into a bitmask or something. Getting a bit out of hand :p